automation/vars/uni04delta.yaml

```diff
@@ -0,0 +1,77 @@
+---
+vas:
+  uni04delta:
+    stages:
+      - path: examples/dt/uni04delta/control-plane/nncp
+        wait_conditions:
+          - >-
+            oc -n openstack wait nncp
+            -l osp/nncm-config-type=standard
+            --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured
+        values:
+          - name: network-values
+            src_file: values.yaml
+        build_output: nncp.yaml
+
+      - path: examples/dt/uni04delta/control-plane
+        wait_conditions:
+          - >-
+            oc -n openstack wait osctlplane controlplane
+            --for condition=Ready
+            --timeout=600s
+        values:
+          - name: network-values
```

Comment:
@jirimacku the field `src_file: values.yaml` is missing.

This caused: https://github.com/openstack-k8s-operators/architecture/actions/runs/9028525856/job/24811285679#step:5:18

Reply:
Fixed